modules/wrapper/src/main/java/com/beust/kobalt/wrapper/Main.java

```diff
@@ -425,6 +432,18 @@ private void downloadImpl(File file, String version) throws IOException {
         }
     }
 
+    private Proxy getProxy() {
+        String http_proxy = System.getenv("http_proxy");
```

Comment:
This was only a first try in order to get something working. And sorry for the variable name. Do you want me to fix it?